templates/tfengine/components/devops/variables.tf

```diff
@@ -12,28 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 {{$props := .__schema__.properties -}}
-{{$missing_admins_group := not (get .admins_group "exists")}}
+{{$adminsGroupProps := $props.admins_group.properties -}}
+{{$projectProps := $props.project.properties -}}
+{{$projectOwnersGroupProps := $projectProps.owners_group.properties -}}
 variable "admins_group" {
   type = object({
-    id           = string
-    {{- if $missing_admins_group}}
     customer_id  = string
-    display_name = string
-    {{- if has .admins_group "description"}}
     description  = string
-    {{- end}}
-    {{- if has .admins_group "owners"}}
+    display_name = string
+    exists       = string
+    id           = string
     owners       = list(string)
-    {{- end}}
-    {{- if has .admins_group "managers"}}
     managers     = list(string)
-    {{- end}}
-    {{- if has .admins_group "members"}}
     members      = list(string)
-    {{- end}}
-    {{- end}}
   })
-  description = {{schemaDescription $props.admins_group.description}}
+  description = <<EOF
+    {{$props.admins_group.description}}
+
+    * customer_id = {{$adminsGroupProps.customer_id.description}}
```

Comment:
Same as the other PR, consider just looping through `$adminsGroupProps` here and adding a "Fields:" line right before to make it clearer what this section of the description is.

Reply:
Same resolution as the other PR, only added Field header